src/test/java/org/folio/rest/impl/FeeFinesAPITest.java

```diff
@@ -16,14 +17,6 @@
 @RunWith(JUnitParamsRunner.class)
 public class FeeFinesAPITest extends ApiTests {
   private static final String REST_PATH = "/feefines";
-  private static final String FEEFINES_TABLE = "feefines";
-  private static final String OWNERS_TABLE = "owners";
-
-  @Before
-  public void setUp() {
-    removeAllFromTable(FEEFINES_TABLE);
-    removeAllFromTable(OWNERS_TABLE);
-  }
```

Comment:
Why did we remove these lines? I guess they delete automatic fees/fines, but they were added here to guarantee that tests are independent. I would rather make one of them to delete everything except automatic types.

Reply:
fixed